Orange/widgets/data/owconcatenate.py

```diff
@@ -115,15 +115,9 @@ def __init__(self):
         cb.disables.append(ibox)
         cb.makeConsistent()
 
-        gui.separator(self.controlArea, 8)
-
-        box = gui.widgetBox(self.controlArea, True, orientation="horizontal",)
-        box.layout().addWidget(self.report_button)
-        gui.separator(box, 20)
-        gui.button(
-            box, self, self.tr("Apply Changes"),
-            callback=self.apply, default=True
-        )
+        gui.separator(self.buttonsArea, 20)
+        gui.button(self.buttonsArea, self, "Apply Changes",
```

Comment:
I thought about it (haven't we also discussed it), but each widget has its special way of arranging the texts, checkboxes and buttons. If we move auto apply to declarations, we would basically move the arguments of the `auto_commit` method to widget's attributes.